### PR TITLE
Expected Scopes of ClientScopesCondition created on Admin UI are not …

### PR DESCRIPTION
**Pull request by tnorimat**
…saved onto ClientScopesCondition.Configuration

Closes #9371

### CODE REVIEW

**Comment by tnorimat** [comment]
This PR is for https://github.com/keycloak/keycloak/pull/9526

**Review by mposolda — DISMISSED** [review]
@tnorimat Thanks!

@andymunro Do you please have a chance to review?

File: upgrading/topics/keycloak/changes-17_0_0.adoc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+= Client Policies Migration : client-scopes
+
+If you used a policy including client-scopes condition and edited JSON document directly, you will need to change "scope" field name in a JSON document to "scopes".
```
Comment (andymunro) [inline upgrading/topics/keycloak/changes-17_0_0.adoc]: Please add "the" after "change"

**Review by andymunro — CHANGES_REQUESTED** [review]
Thanks, @tnorimat.  Just one small change.

File: upgrading/topics/keycloak/changes-17_0_0.adoc (hunk 1 above)
Comment (tnorimat) [reply upgrading/topics/keycloak/changes-17_0_0.adoc]: Yes, I've fixed it as you've suggested.

**Comment by tnorimat** [comment]
@andymunro Thank you.  I've fixed it as you've suggested.

**Review by andymunro — APPROVED** [review]
Thanks, @tnorimat Looks good.

**Comment by mposolda** [comment]
@andymunro Thanks for the review!

**Review by mposolda — APPROVED** [review]

**Pull request merged**